clerk/clerk.go

```diff
@@ -13,26 +13,26 @@ import (
 	"time"
 )
 
-const version = "1.21.0"
+const version = "1.23.0"
```

Comment:
❓ Why not `1.22.0`?

Reply:
I come from the future. :stuck_out_tongue: :timer_clock: :man_scientist: